mod1/src/file-2.xoo

```diff
@@ -0,0 +1,7 @@
+// dummy line 1
+// dummy line 2
+// dummy line 3
+// dummy line 4
+// dummy line 5
+// dummy line 6
+// violate HasTag rule: xoo on line 7
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:34029/organizations/org5/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:34029/project/issues?id=sample&pullRequest=6833&issues=AWlNI7Tdk4hMdV0eO1Fv&open=AWlNI7Tdk4hMdV0eO1Fv)

[ISSUE_KEY]: AWlNI7Tdk4hMdV0eO1Fv